butane_core/tests/adb.rs

```diff
@@ -373,10 +373,10 @@ async fn add_table_fkey_back_reference(conn: ConnectionAsync) {
             vec![
                 "CREATE TABLE a (",
                 "fkey INTEGER NOT NULL PRIMARY KEY,",
-                "FOREIGN KEY (fkey) REFERENCES b(id)",
+                "FOREIGN KEY (fkey) REFERENCES b(\"id\")",
```

Comment:
[sqlparser](https://crates.io/crates/sqlparser) might be using this list now
https://www.postgresql.org/docs/current/sql-keywords-appendix.html

looks like we might be able to improve our quoting by using the finer grained lists at https://github.com/apache/datafusion-sqlparser-rs/blob/main/src/keywords.rs ; iirc they have been added since we added this feature.